### DISCUSSION TITLE
PCL2新闻网站好像被净网行动暂时封了

### DISCUSSION THREAD

**Discussion by mc-124** [Meloong-Git/PCL#2585] [提问] [answered]
![Plain Craft Launcher　 2023_8_23 20_42_12](https://github.com/Hex-Dragon/PCL2/assets/138775126/aa1808ab-c3a0-4b9d-80bb-2799fc847e21)
现在看不到MC新闻主页

**Comment by WForst-Breeze** [comment]
![image](https://github.com/Hex-Dragon/PCL2/assets/110760354/9ab93975-545c-4c8a-a19e-80f3496a996d)
我清理了缓存之后刷新主页，一切正常，应该是你个人网络的问题……？使用内置的预设主页和手动输入地址获取主页都可以正常访问（

**Comment by zkitefly** [comment]
@Light-Beacon 看看？

**Comment by iPodtouch6557** [comment]
个人问题吧……我也一切正常
<img width="704" alt="捕获4" src="https://github.com/Hex-Dragon/PCL2/assets/118656946/5c5aab6e-6446-40e0-9a3e-2acae6fbe061">

**Comment by MoYuan-CN** [comment]
Woc 换网址了？
我这两天也看不到新闻主页了

**Comment by Light-Beacon** [comment]
我这看着没问题啊...

**Reply by MoYuan-CN** [reply]
预设居然是错误的 https://news.bugjump.net
不过.... ![image](https://github.com/Hex-Dragon/PCL2/assets/109946426/931b6f87-6f3e-418e-831f-7a45a1eff8b0) emmm....

**Reply by Light-Beacon** [reply, marked as answer]
> 预设居然是错误的 https://news.bugjump.net
> 
> 不过.... ![image](https://github.com/Hex-Dragon/PCL2/assets/109946426/931b6f87-6f3e-418e-831f-7a45a1eff8b0) emmm.... 

这个其实搞了另一个地址，并且比原来方案优化了一下照理来说应该能访问的话比原来的快，如果不能访问还是使用原来的吧

**Reply by LTCatt** [reply]
@Light-Beacon 如果你之后确认这个需要回滚的话那交个issue我下个版本改吧……不然该忘了 =。=
